week1/Daily_bike_trips_prediction.Rmd

```diff
@@ -0,0 +1,162 @@
+---
+title: "predict_citibike.Rmd"
+author: "Ahuva Bechhofer"
+date: "6/17/2021"
+output: html_document
+---
+
+```{r setup, include=FALSE}
+knitr::opts_chunk$set(echo = TRUE)
+# load some packages that we'll need
+
+library(tidyverse)
+library(scales)
+library(modelr)
+library(stats)
+
+
+#  white backgrounds on our plots
+theme_set(theme_bw())
+options(repr.plot.width=4, repr.plot.height=3)
+
+
+# load the Data file 
+trips_per_day <- read_tsv('trips_per_day.tsv')
+holidays <- read.delim("holidays_2014.txt",header = FALSE, sep = ",", dec = ".")

```

Comment:
check out the package `library(Holidays)` - seems to provide a package for selecting US holidays! (saw other students use this)